packages/webpack-loaders/example/build.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+
+const webpackLoaders = require("..");
+const Interlock = require("interlock");
+const css = require("interlock-css");
+const autoprefixer = require("autoprefixer");
+
+Error.stackTraceLimit = Infinity;
+
+const ilk = new Interlock({
+  srcRoot: __dirname,
+  destRoot: path.join(__dirname, "dist"),
+
+  entry: {
+    "./app/example.js": "example.bundle.js"
+  },
+
+  pretty: true,
+
+  plugins: [
+    css({
+      mode: "object",
+      plugins: [autoprefixer]
+    }),
+    webpackLoaders({
+      loaders: [
+        {
+          test: /\.json$/,
+          loader: require.resolve("json-loader")
+        },
+        {
+          test: /\.scss$/,
+          loader: require.resolve("sass-loader"),
+          transpileTarget: "css"
```

Comment:
Interlock has the concept of an input module type.  Is that was this is, essentially?